DirectProgramming/DPC++FPGA/Tutorials/Features/ac_fixed/src/ac_fixed.cpp

```diff
@@ -0,0 +1,200 @@
+// clang-format off
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/ext/intel/ac_types/ac_fixed.hpp>
+#include <sycl/ext/intel/ac_types/ac_fixed_math.hpp>
+// clang-format on
```

Comment:
Why do we need these clang-format on/off comments here? 
I don't see us using these in the ac_int tutorial for example